enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -169,7 +169,16 @@ CCO should consider adding the above, three-part test as a utility function expo
 **HyperShift changes**: Include Cloud Credential Operator with token-aware mode (see above). Allows for processing of 
 CredentialsRequest objects added by operators. 
 
-The CCO is expected to run on the control plane NS, not on the customer's worker nodes, and not be visible to the customer. However if CredentialsRequests are created on the worker nodes, the CCO will process them and yield Secrets as described elsewhere in this document.
+The CCO is expected to run on the control plane NS, not on the customer's worker nodes, and not be visible to the customer. We propose that multiple 
+instances of CCO be installed on the control plane each consuming the 
```

Comment:
this phrasing is confusing.  You're installing multiple instances on the management cluster, but it's still just one instance per hosted control plane (i.e. one per hosted cluster)

Reply:
you might want to mention that it will run in the hosted control plane's NS on the management cluster, but othewise yes, better.